edx-platform/pearson-learninghub-theme/conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -3482,3 +3482,6 @@ msgstr ""
 msgid "Create Account"
 msgstr ""
 
+#: edx-platform/pearson-pols-theme/lms/templates/student_account/register.underscore:46
+msgid "Create a new account here"
```

Comment:
What about the interpolated string? @luismorenolopera ...We should have something like [this](https://github.com/proversity-org/proversity-openedx-themes/blob/613663e9c0637eda722bda8d33006b98f013e9d3/edx-platform/pearson-learninghub-theme/conf/locale/en/LC_MESSAGES/django.po#L3445)

Reply:
It's not necessary to extract the translation in this PR, but we must check  the interpolation of the string and make sure it's working as expected.